Build/GameSettings.cc

```diff
@@ -186,7 +186,7 @@ void InitGameOptions()
 	gGameOptions.fSciFi						 = TRUE;
 	gGameOptions.ubDifficultyLevel = DIF_LEVEL_EASY;
 	//gGameOptions.fTurnTimeLimit		 = FALSE;
-	gGameOptions.fIronManMode			 = FALSE;
+	gGameOptions.ubGameSaveMode			 = DIF_CAN_SAVE;
```

Comment:
That had me a bit confused as well, however is done in the other Options as well. Basically GIO are the definitions for the Game Init Option Screen buttons and DIF are the saved difficulty variables. I remember finding a reason it was actually done that way but don't remember what it was.